pipeline/alert.go

```diff
@@ -963,6 +968,70 @@ type SensuHandler struct {
 	*AlertNode
 }
 
+// Send the alert to Pushover.
+//
```

Comment:
How does one get a token to Pushover? Should we add steps here on how to do that?